js/gallery.js

```diff
@@ -0,0 +1,62 @@
+const _galleryContent = `
```

Comment:
I'm not sure why your variables are prefixed with an underscore. Maybe you can help clarify this?

Reply:
Just my way of naming things at times. It's a habit.

Reply:
It's misleading. When variables are prefixed with an underscore, that usually means that they are private.

You should be deliberate about naming. It's a very vital part of building scalable and usable applications.